### PR TITLE
Multiple cities weather

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -1,55 +1,90 @@
...
+    getCityLocatin({ name: "Tehran", countryCode: "IR" });
+  } else if (selectedCityName.value === "stockholm") {
+    getCityLocatin({ name: "Stockholm", countryCode: "SE" });
+  }
+});
+
+async function getCityLocatin(object) {
+  const fetchedData = await fetch(
+    `https://api.openweathermap.org/geo/1.0/direct?q=${object.name},${object.countryCode}&limit=1&appid=e0cb24360c821c3571a49c9d05be0fb1`
+  );
```
Comment (blacksrc) [inline assets/js/main.js]:
```suggestion
async function getCityLocatin(city) {
  const fetchedData = await fetch(
    `https://api.openweathermap.org/geo/1.0/direct?q=${city.name},${city.countryCode}&limit=1&appid=e0cb24360c821c3571a49c9d05be0fb1`
  );
```
Or 
```suggestion
async function getCityLocatin({ name, countryCode }) {
  const fetchedData = await fetch(
    `https://api.openweathermap.org/geo/1.0/direct?q=${name},${countryCode}&limit=1&appid=e0cb24360c821c3571a49c9d05be0fb1`
  );
  ```

File: assets/js/main.js [hunk 2]
```diff
@@ -1,55 +1,90 @@
...
+  const selectedCityLocation = await fetchedData.json();
+
+  const lat = selectedCityLocation[0].lat;
+  const lon = selectedCityLocation[0].lon;
```
Comment (blacksrc) [inline assets/js/main.js]: Use object destructuring here.

File: assets/js/main.js [hunk 3]
```diff
@@ -1,55 +1,90 @@
...
+  getWeather(lat, lon);
+}
+
+async function getWeather(lat, lon) {
+  const result = await fetch(
+    `https://api.openweathermap.org/data/2.5/forecast?lat=${lat}&lon=${lon}&units=metric&appid=e0cb24360c821c3571a49c9d05be0fb1`
+  );
+  const resultJson = await result.json();
+  const cityName = document.getElementById("city-name");
+  const { city, list } = resultJson;
```
Comment (blacksrc) [inline assets/js/main.js]:
```suggestion
  const { city, list } = await result.json();
  const cityName = document.getElementById("city-name");

File: assets/js/main.js [hunk 4]
```diff
@@ -1,55 +1,90 @@
...
+  } else if (selectedCityName.value === "tehran") {
+    getCityLocatin({ name: "Tehran", countryCode: "IR" });
+  } else if (selectedCityName.value === "stockholm") {
+    getCityLocatin({ name: "Stockholm", countryCode: "SE" });
+  }
+});
+
+async function getCityLocatin(object) {
+  const fetchedData = await fetch(
+    `https://api.openweathermap.org/geo/1.0/direct?q=${object.name},${object.countryCode}&limit=1&appid=e0cb24360c821c3571a49c9d05be0fb1`
```
Comment (blacksrc) [inline assets/js/main.js]: Define a variable on top of the file as `appId` and use that in the functions.

File: assets/js/main.js [hunk 5]
```diff
@@ -1,55 +1,90 @@
...
+
+  const lat = selectedCityLocation[0].lat;
+  const lon = selectedCityLocation[0].lon;
+
+  getWeather(lat, lon);
+}
+
+async function getWeather(lat, lon) {
+  const result = await fetch(
+    `https://api.openweathermap.org/data/2.5/forecast?lat=${lat}&lon=${lon}&units=metric&appid=e0cb24360c821c3571a49c9d05be0fb1`
```
Comment (blacksrc) [inline assets/js/main.js]: Define a variable on top of the file as `appId` and use that in the functions.

File: assets/js/main.js (hunk 2 above)
Comment (hedita) [reply assets/js/main.js]: Done.

File: assets/js/main.js (hunk 5 above)
Comment (hedita) [reply assets/js/main.js]: Done.

File: assets/js/main.js (hunk 4 above)
Comment (hedita) [reply assets/js/main.js]: Done.

File: assets/js/main.js [hunk 6]
```diff
@@ -1,55 +1,100 @@
...
+
+  const iconId = document.getElementById("current-icon");
+  const icon = list[0].weather[0].icon;
+  const iconImage = `http://openweathermap.org/img/wn/${icon}@2x.png`;
+  iconId.innerHTML = `<img src=${iconImage} />`
+
+  const todayIcon = document.getElementById("today-icon");
+  todayIcon.innerHTML = `<img src=${iconImage} />`
+
+  const tomorrowIcon = document.getElementById("tommorow-icon");
```
Comment (takrami) [inline assets/js/main.js:73]: src is the same as today's icon. so you are getting the same icons for all days. please fix this.

Comment (hedita) [reply assets/js/main.js:73]: Fixed.

**Review by takrami — APPROVED** [review]

**Pull request merged**